GameData/RealismOverhaul/RealPlume_Configs/Squad/sepMotor1.cfg

```diff
@@ -1,4 +1,4 @@
-@PART[sepMotor1]:BEFORE[RealPlume]
+@PART[sepMotor1|sepMotorSmall|sepMotorLarge|ROSmallSpinMotor]:BEFORE[RealPlume]
```

Comment:
The plume size is the same on all of these parts with this patch. The plume should be scaled to match the scale of the specific part